elastalert/loaders.py

```diff
@@ -137,10 +145,7 @@ class RulesLoader(object):
     jinja_environment = Environment(loader=FileSystemLoader(""))
 
     def __init__(self, conf):
-        # schema for rule yaml
-        self.rule_schema = jsonschema.Draft7Validator(
-            yaml.load(open(os.path.join(os.path.dirname(__file__), 'schema.yaml')), Loader=yaml.FullLoader))
```

Comment:
Chasing down unclosed streams from the `yaml` library and came across this one.  Likely not a big issue, but an easy fix.